data/scripts/movements/quests/killing_in_the_name_of/boss.lua

```diff
@@ -309,7 +309,7 @@ local bosses = {
 	},
 	[3261] = {
 		bossName = 'the horned fox',
-		storage = 17522,
+		storage = 14896,
```

Comment:
i've changed this file in my fix, this changes will be not needed

Reply:
Awesome

Reply:
Is it now up to me to remove this file from the pull request?

Reply:
revert change as it was and it all be good